src/database/queries/get_category.js

```diff
@@ -0,0 +1,7 @@
+const dbConnection = require('../db_connection')
+
+const getCategory = () => dbConnection
+  .query('select * from category')
```

Comment:
you should Capitalize the query command like `SELECT * FROM category`

Reply:
i change it to capitalize